src/app/storage.ts

```diff
@@ -0,0 +1,70 @@
+import { Storage } from '@ionic/storage';
+import localforage from 'localforage';
+
+// INCREMENT THIS whenever you alter the database
+let currentVersion = 0;
```

Comment:
I'm a bit confused by the version numbering here. Shouldn't `currentVersion` be something greater than 0?  That is, before we start, the version is `0`, and when we end the version is `0`